## Imports

In [2]:
import os 

## Exploring the folder structure
We check that our data folders are visible from the notebook.

In [ ]:
print(os.listdir(".")) 

['.DS_Store', '.gitignore', 'test', 'train', 'val', 'zoidberg.ipynb']


## Exploring the Data
We count how many images are in each folder to understand our dataset.

In [16]:
print(len(os.listdir("test/NORMAL")))
print(len(os.listdir("test/PNEUMONIA")))
print(len(os.listdir("train/NORMAL")))
print(len(os.listdir("train/PNEUMONIA")))
print(len(os.listdir("val/NORMAL")))
print(len(os.listdir("val/PNEUMONIA")))


234
390
1342
3876
9
9


We notice a class imbalance in the training set: 1342 NORMAL vs 3876 PNEUMONIA. 
This could bias our model toward predicting pneumonia.